src/components/action-buttons.component.tsx

```diff
@@ -55,14 +58,31 @@ const ActionButtons: React.FC<ActionButtonsProps> = ({
     medicationRequestStatus === MedicationRequestStatus.active &&
     mostRecentMedicationDispenseStatus !== MedicationDispenseStatus.declined;
 
+  const DesignName = () => {
+    const styles = { paddingLeft: "20px", color: "gray", fontWeight: 500 };
+
+    if (patient) {
+      return (
+        <span style={styles}>
+          {`${patient.name[0].given[0]} ${patient.name[0].family} — #${patient.identifier[0].value}`}
+        </span>
```

Comment:
this seems simple enough to get rid of the rest of the lines after extracting the styles